--- lazr.restfulclient-0.11.2/setup.py.orig	2011-06-10 14:23:56.000000000 -0500
+++ lazr.restfulclient-0.11.2/setup.py	2011-06-10 14:24:05.000000000 -0500
@@ -55,12 +55,6 @@
         'src/lazr/restfulclient/NEWS.txt'),
     license='LGPL v3',
     install_requires=[
-        'httplib2',
-        'lazr.authentication',
-        'oauth',
-        'setuptools',
-        'wadllib>=1.1.4',
-        'wsgi_intercept',
         ],
     url='https://launchpad.net/lazr.restfulclient',
     download_url= 'https://launchpad.net/lazr.restfulclient/+download',
